manual/manual.tex

```diff
@@ -114,12 +115,29 @@
   \begin{abstract}
 % This list of contributors is based on the hg log. If you make commits please
 % add your name in alphabetical order.
-The primary authors of this document are Matthew Grosvenor and Adam Walker,
-with contributions from Adrian Danis, Andrew Boyton, Anna Lyons, David Greenaway, Etienne
-Le Sueur, Gernot Heiser, Gerwin Klein, Godfrey van der Linden, Kevin
-Elphinstone, Matthew Fernandez, Matthias Daum, Michael von Tessin, Peter Chubb,
-Simon Winwood, Thomas Sewell, Timothy Bourke and Toby Murray. All authors 
-and contributors can be contacted at firstname.lastname@data61.csiro.au.
+The primary authors of this document are Matthew~Grosvenor and Adam~Walker,
+with contributions from
+Adrian~Danis,
+Andrew~Boyton,
+Anna~Lyons,
+Axel~Heider,
+David~Greenaway,
+Etienne~Le~Sueur,
+Gernot~Heiser,
+Gerwin~Klein,
+Godfrey~van~der~Linden,
+Kevin~Elphinstone,
+Matthew~Fernandez,
+Matthias~Daum,
+Michael~von~Tessin,
+Peter~Chubb,
+Simon~Winwood,
+Thomas~Sewell,
+Timothy~Bourke and
+Toby~Murray.
+\\
+\\
+All authors and contributors can be contacted via the seL4 Foundation.
```

Comment:
Would rephrase that, we probably collectively have a private email address for everyone in here, but I don't think everyone in the list wants to be contactable via the foundation.

Maybe instead we should add a link to discourse/mailing list for contact?

Reply:
I don't have any preference what to put there and I'm open to suggestions. Basically I just removed the sentence about TS e-mail, because I was assuming this is no longer really correct. I'll change this to discourse and the mailing list then.